File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/utils/Links.java
package ch.rmy.android.statusbar_tacho.utils;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

public class Links {

    private static final String GITHUB_URL = "https://github.com/Waboodoo/Status-Bar-Tachometer";

    public static void openGithub(Context context) {
        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(GITHUB_URL));
        context.startActivity(intent);
    }

}

File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/utils/Links.kt
package ch.rmy.android.statusbar_tacho.utils

import android.content.Context
import android.content.Intent
import android.net.Uri

object Links {

    private val GITHUB_URL = "https://github.com/Waboodoo/Status-Bar-Tachometer"

    fun openGithub(context: Context) {
        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(GITHUB_URL))
        context.startActivity(intent)
    }

}
